Add instructions to update ffmpeg on Ubuntu 18.04 (#553)
home-assistant/developers.home-assistant: docs/development_environment.md

From Windows Store install Ubuntu.

#### Additional steps for Ubuntu 18.04

> If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest ffmpeg version. When you are promted to acknowledge the displayed information you can press ENTER to proceed.

Reviewer: ```suggestion
If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version. When you are prompted to acknowledge the displayed information you can press ENTER to proceed.
```

Reviewer: This is the developer documentation, if feel that this line:

> When you are prompted to acknowledge the displayed information you can press ENTER to proceed.

Might be a bit overkill :)

Author: I removed the line. I guess there are not that many people using a complex setup like WSL who would need the additional encouragement to press ENTER. 😅

From Windows Store install Ubuntu.

#### For those already running Ubuntu 18.04 on WSL

> If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version.

Reviewer 2: Very small tweak otherwise I think it looks great.
```suggestion
If are running Ubuntu 18.04 on Windows Subsystem for Linux some additional steps are needed to install the latest FFmpeg version.
```

Author: Implemented the changes with the missing "you"
